packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -401,6 +402,13 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
 
       yield put(A.createOrderLoading())
 
+      yield put(
+        cacheActions.setLastUnusedAmount({
+          amount: convertBaseToStandard('FIAT', input.amount),
```

Comment:
There is a constant FIAT defined, not sure if the place where it was defined is correct but it would be great to use that instead of a hardcoded string

Reply:
This is typed for a string, I think it should keep this way, as it can be `FIAT` or any other `coin`